crates/saya/README.md

```diff
@@ -61,20 +63,21 @@ To ensure we've the exact same result, Saya must run the same version (or at lea
 In new Cairo VM version, there are breaking changes as mentioned in the release not, which implies a bump of Cairo VM for Katana and at the same time we could bump to cairo `2.5.0`.
 However, papyrus and blockifier which we depend on are still in `-dev` version, where also some breaking changes must be addressed.
 
-* Cairo VM (currently dojo is using 0.8, and others are in 0.9)
-* Blockifier (uses Cairo VM and cairo-lang `-dev`)
-* Papyrus (used by blockifier and use blockifier and cairo-lang `-dev`)
-* cairo-lang (we should support `2.5` now)
-* scarb (breaking changes between 2.4 and 2.5 to be addresses, not required to only build saya and SNOS)
+- Cairo VM (currently dojo is using 0.8, and others are in 0.9)
+- Blockifier (uses Cairo VM and cairo-lang `-dev`)
+- Papyrus (used by blockifier and use blockifier and cairo-lang `-dev`)
+- cairo-lang (we should support `2.5` now)
+- scarb (breaking changes between 2.4 and 2.5 to be addresses, not required to only build saya and SNOS)
 
 ## Local Testing
 
 ```bash
 cargo run -r -p katana # Start an appchain
 cargo run -r -p sozo -- build --manifest-path examples/spawn-and-move/Scarb.toml
-cargo run -r -p sozo -- migrate --manifest-path examples/spawn-and-move/Scarb.toml # Make some transactions
-cargo run -r --bin saya -- --rpc-url http://localhost:5050 # Run Saya
+cargo run -r -p sozo -- migrate apply --rpc-url http://localhost:5050 --manifest-path examples/spawn-and-move/Scarb.toml # Make some transactions
+cargo run -r --bin saya -- --rpc-url http://localhost:5050 --registry 0x217746a5f74c2e5b6fa92c97e902d8cd78b1fabf1e8081c4aa0d2fe159bc0eb --world ... # Run Saya
 ```
+
```

Comment:
Mention here that the local testing is currently only supported on `amd64` arch.

Reply:
fixed